rotkehlchen/tests/api/test_manually_tracked_balances.py

```diff
@@ -261,8 +268,13 @@ def test_add_manually_tracked_balances_no_price(rotkehlchen_api_server):
         result = outcome['result']
     else:
         result = assert_proper_response_with_result(response)
+    expected_balances = []
+    for new_id, balance in zip(range(1, len(balances) + 1), balances):
```

Comment:
You can achieve the same in a simpler way with `enumerate`

```suggestion
    for new_id, balance in enumerate(balances, start=1):
```